en_US/network/emqx-mqtt-tls.md

```diff
@@ -1,53 +1,114 @@
+2. **Confidentiality**: Each session will be encrypted using the session key negotiated by both parties. No third party can know the communication content, so even if the session key is compromised, it does not affect the security of other sessions.
+3. **Integrity**: The possibility of data being tampered with in encrypted communication is extremely low.
 
-  proxy_protocol = false
-  proxy_protocol_timeout = 3s
+## Enable SSL/TLS Connection via Configuration File
 
-  ssl_options.keyfile = "etc/certs/key.pem"
-  ssl_options.certfile = "etc/certs/cert.pem"
-  ssl_options.cacertfile = "etc/certs/cacert.pem"
+::: tip Prerequisite
 
-  # to verify client certs
-  ssl_options.verify = verify_peer
+SSL/TLS certificates are available.
 
-  ssl_options.versions = ["tlsv1.3", "tlsv1.2", "tlsv1.1", "tlsv1"]
-  tcp_options.backlog = 1024
-  tcp_options.buffer = 4KB
-}
-```
+EMQX only provides a set of SSL/TLS certificates (located in the `etc/certs` directory of the installation package) for testing purposes and enables SSL/TLS connections on port `8883`. When used in a production environment, reliable certificates signed by a trusted CA should be used. For information on how to apply for relevant certificates, see [Extended Reading: Obtain SSL/TLS Certificates](#Extended-reading-obtain-ssl-tls-certificates).
 
-::: tip
-The `key.pem`,` cert.pem`, and `cacert.pem` under the default directory of `etc/certs` are self-signed certificates generated by EMQX. Therefore, when testing with a client that supports TLS, you need to configure the above CA certificate `etc/certs/cacert.pem` to the client.
+:::
 
-For production use, securely issued certificates must be used.
+1. Move the SSL/TLS certificate file to the `etc/certs` directory of EMQX.
 
-:::
+2. Open the configuration file `emqx.conf` (located in either the `./etc` or `/etc/emqx/etc` directory depending on your installation method). Modify the `listeners.ssl.default` configuration group, replace the certificate with your own certificate, and add `verify = verify_none`:
+
+   ```bash
+   listeners.ssl.default {
+     bind = "0.0.0.0:8883"
+     max_connections = 512000
+     ssl_options {
+       # keyfile = "etc/certs/key.pem"
+       keyfile = "etc/certs/server.key"
+       # certfile = "etc/certs/cert.pem"
+       certfile = "etc/certs/server.crt"
+       # cacertfile = "etc/certs/cacert.pem"
+       cacertfile = "etc/certs/rootCA.crt"
+   
+       # Peer verification not enabled
+       verify = verify_none
+     }
+   }
+   ```
+
+   Now you have completed the SSL/TLS one-way authentication configuration on EMQX. One-way authentication only ensures that the communication is encrypted but cannot verify the client's identity.
+
+   If you need to enable two-way authentication, add the following configuration to the `listeners.ssl.default` configuration group:
```

Comment:
what is two-way authentication and benefits

Reply:
Added the descriptions at the beginning.